operator_ui/src/components/Bridges/List.tsx

```diff
@@ -74,23 +73,23 @@ type RouteProps = RouteComponentProps<{
 type Props = OwnProps & RouteProps
 
 // FIXME - remove unused export?
-export const BridgeList = useHooks<Props>(props => {
+export const BridgeList = (props: Props) => {
   const {
     bridges,
     bridgeCount,
+    error,
     fetchBridges,
-    pageSize,
     fetching,
-    error,
+    match,
+    pageSize,
   } = props
```

Comment:
What do you think about `destructuring` props arguments here? for example:
`export const BridgeList = ({bridges, bridgeCount, ...}) => {}`